extension/src/components/OpenLoans.tsx

```diff
@@ -20,7 +20,7 @@ import { LoanDetails } from "../models";
 import Btc from "./bitcoin.svg";
 import Usdt from "./tether.svg";
 
-const error = Debug("openloans:error");
+const debug = Debug("openloans:error");
```

Comment:
You do like renaming these ha? :D

Reply:
That was not on purpose - rebasing issue 😅 - I'll fix it